string type
float64[] argv
---
string res